ext/drpttracer/drpttracer.c

```diff
@@ -517,47 +599,67 @@ drpttracer_end_tracing(IN void *drcontext, INOUT void *tracer_handle,
         return DRPTTRACER_ERROR_FAILED_TO_STOP_TRACING;
     }
 
+    /* Check if the PT data in the aux ring buffer is overwritten. */
+    uint64_t pt_size = handle->header->aux_head - handle->header->aux_tail;
+    if (pt_size > handle->header->aux_size) {
+        ERRMSG("the size of PT trace data(%" PRIu64
```

Comment:
DR provides cross-platform format string codes (e.g. `UINT64_FORMAT_CODE`) which we generally use in existing code.  That was provided before these new C99 codes were implemented in Visual Studio.  Maybe it is ok to use them now since we're on VS2019 which I believe (finally) implements all of C99?

Reply:
Done.